app/code/Scandiweb/ContentTypes/view/frontend/templates/content-type/featured-block.phtml

```diff
@@ -26,17 +26,42 @@ $secondCtaLink = $block->getSecondCtaLink();
 
 // Visual
 $isVisualOnLeft = $block->getVisualContentPosition() === 'left';
-$image = $block->getImage();
+$visualType = $block->getVisualType();
+$image = $block->getImage($visualType);
+$videoUrl = $block->getVisualContentVid();
+$placeHolderType = '';
+
+//Visual Type Conditional Statements
+$isImageExist = $visualType === 'image' && $image;
+$isVideoExist = $visualType === 'video' && $videoUrl;
+
+if ($isVideoExist) {
+    $placeHolderType = $block->getPlaceholderType();
+    $imagePlaceholder = $placeHolderType === 'uploader' ?
+        $image['url'] : $block->getVisualContentVideoPlaceholderUrl();
+}
```

Comment:
No need even to pass $visualType, it is available in php block as getData(...). So if can please simplify a bit logic out here, to use as less as possible variables.